CHANGELOG.md

```diff
@@ -126,7 +136,7 @@ Fixed
 - Fixed `guesseed` -> `guessed` typo in `archive` state. (#53480)
 - Fixed error with incorrect import statement masking real import error. (#53508)
 - Added some error handling around missing results from external returners. (#53517)
-- Changed to match repo paramter against repo name on `salt-run git_pillar.update`, so remote name can be used instead of full remote URL. (#53622)
+- Changed to match repo paramter against repo name on `salt-run git_pillar.update`, so remote name can be used instead of full remote URL. (#56605 - Master port of#53622)
```

Comment:
No need to spell out the master port.

When someone goes to check that issue #, they'll see that there is a master port.